pages/Documents/MessagingChannels/ConnectToMessaging/api.md

```diff
@@ -0,0 +1,254 @@
+- C2M does not have any limitations on the message size while sending messages to twilio or other channels. However a large message may translate to more than one message when the recipient receives it.
+- Example: A message of more than 140 characters will be divided into two messages and sent to recipients.
+ 
+<strong>What is the lifespan of the app JWT? When we do need to get a new JWT, do we have to first make the call to LivePerson Domain API in order to get the sentinel service domain, or is that domain consistent enough that we can hard code that in?</strong>
+- An APP JWT expiration time is 1 hour from the time it is created. To get an app JWT from sentinel API, a call to domain api has to be made to get the sentinel api domain. This domain can be cached for some duration. We expect the domain to change in very rare cases. It’s still recommended that cache duration should not be more than 1 day.
+
+- When using app JWT to call C2M API, a response below indicates the jwt is expired and new app jwt to be obtained from sentinel api.
+	```json
+	{
+		"code": 0,
+		"message": "jwt expired"
+	}
+	```
+
+<strong>Do we need any other JWT other than APP JWT e.g. Consumer JWT?</strong>
+C2M service does not create or consume consumer JWT or other JWT except APP JWT. C2M API consumes AppJWT created from provided clientId and Secret for authentication.
+
+<strong>What should the authentication header look like, is the bearer token the only thing required even in production usage? Do we need to include our ConsumerKey/Secret or our AccessToken/Secret that we use in the 1.0 API at all, or any other information?</strong>
+App Jwt will be consumed as Bearer Token. No other key, secret or token will be consumed by C2M Messaging api.
+
+<strong>How does C2M 2.0 api provide status of message e.g. success/failure of delivery ?</strong>
+The C2M 2.0 campaign api is asynchronous meaning that the success and failure of a message to a recipient is noted only when the recipient is picked from the C2M internal queue and a message is sent as per pre configured message rate. The [conversations](#conversations-api-example-request-and-response) api will provide the status of recipients tied to the campaign created.
+
+<strong>Of the error cases described above, which of those errors should we consider "retry-able"? For example, a bad request due to a missing field is not retry-able because it will just always fail, but a case where one of the downstream services was temporarily unavailable could warrant a retry. Which error cases that we could get back from the /campaigns endpoint are retryable and how should we handle a retry to avoid sending duplicate messages to a customer?</strong>
+C2M Messaging service has retry mechanism internally on dependent services to reduce failures due to transient errors.
```

Comment:
mention the rate limit 429 and recommended retry delay here?

Reply:
I'll put this explanation below the rate limit section.